SETUP.md

```diff
@@ -79,3 +79,28 @@ Please submit translations via [Transifex][transifex].
 [1]: https://owncloud.org/about/contributor-agreement/
 [2]: http://opensource.org/licenses/MIT
 [3]: https://owncloud.org/contribute/iOS-license-exception/
+
+## Local ownCloud server for testing
+
+You can easily fire up an ownCloud server on your Ma using Docker.
+
+1. [Install Docker](https://docs.docker.com/docker-for-mac/)
```

Comment:
I think we should not point to macOS specific URL, but the generic one -> [Install docker](https://docs.docker.com/install/#supported-platforms), because the contributor/tester maybe is using another OS.

Same for the `Terminal.app` section.

Reply:
Didn't test Win and Linux. Cover other platforms later?